test/integration/testExecute.js

```diff
@@ -213,14 +218,37 @@ describe('Execute test - variant', function ()
   var testCases =
     [
       {
-        name: 'raw xml',
+        name: 'xml_single_element',
+        type: 'XML',
+        fileExtension: '.xml',
+        sampleData: `<${TEST_HEADER}>${TEST_XML_VAL}</${TEST_HEADER}>`
+      },
+      {
+        name: 'xml_single_element_with_attribute',
+        type: 'XML',
+        fileExtension: '.xml',
+        sampleData: `<${TEST_HEADER} ${TEST_ATTRIBUTE_NAME}=${TEST_ATTRIBUTE_VALUE}>${TEST_XML_VAL}</${TEST_HEADER}>`,
+        ignoreAttributes: false
+      },
+      {
+        name: 'xml_with_attribute_and_custom_parser',
```

Comment:
maybe we can move the xml tests to separate test group and just check that given XML converts to specific json? then we are able to write more specific cases and remove constants

Reply:
Let's add more unit tests

Reply:
I've add 'xml_with_attribute_and_custom_parser' test case